common/flatpak-installation.c

```diff
@@ -1130,7 +1130,7 @@ flatpak_installation_list_installed_refs_for_update (FlatpakInstallation *self,
       if (state == NULL)
         {
 
-          state = flatpak_dir_get_remote_state_optional (dir, remote_name, FALSE, cancellable, error);
+          state = flatpak_dir_get_remote_state_optional (dir, remote_name, FALSE, cancellable, NULL);
```

Comment:
Or do we need to pass `NULL` for `cancellable` too? I'm not clear on the semantics for `GCancellable`.

Reply:
It looks to me like every place where error is used in that function has a return right after it, though. Did you observe any error pileup happening in the wild ?

Reply:
Right below this there's `if (state == NULL) continue;` so on the next iteration the error could be overwritten. I didn't see it in the wild though.

Reply:
oh, if it continues, it should probably do the same thing we do with local_error in the other loop

Reply:
Makes sense, changed it to print a debug message.